solidity-plugin/src/main/java/org/sonarsource/solidity/SolidityRuleContext.java

```diff
@@ -43,6 +43,7 @@ public void addIssue(Token start, Token stop, String reportMessage, String exter
 
   @Override
   public void addIssueOnFile(String reportMessage, String externalRuleKey) {
+    System.out.println("AAAAAAAAAAAAAAAAAAA");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=41&issues=AWSJ8a3Nl6vpliCeT5VF&open=AWSJ8a3Nl6vpliCeT5VF)

[ISSUE_KEY]: AWSJ8a3Nl6vpliCeT5VF